Frends.AzureBlobStorage.ReadBlob/Frends.AzureBlobStorage.ReadBlob/Definitions/Enums.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.ComponentModel;
+#pragma warning disable 1591
+namespace Frends.AzureBlobStorage.ReadBlob
+{
+    [DefaultValue("UTF-8")]
```

Comment:
You don't need this line. Default value should be set in the parameter class, not in the Enum.